Reconstruction/RecCalorimeter/src/components/CreateCaloCells.cpp

```diff
@@ -184,7 +165,7 @@ StatusCode CreateCaloCells::prepareEmptyCells(std::vector<fcc::CaloHit*>& caloCe
   // Get the total number of active volumes in the geometry
   auto highestVol = gGeoManager->GetTopVolume();
   // Substract volumes with active material which are not readout (e.g. ECAL: bath volume)
-  unsigned int numLayers = det::utils::countPlacedVolumes(highestVol, m_activeVolumeName)-m_numVolumesRemove;
+  unsigned int numLayers = det::utils::countPlacedVolumes(highestVol, m_activeVolumeName);
```

Comment:
Sorry, the comment should have been removed as well.
Yes, we fixed the detector and now sensitive layers have a unique volume name so that temporary fix is no longer necessary.